### PR TITLE
chore(bigtable): update quickstart to use preferred method

### LINKED ISSUE #7231: Update bigtable quickstart to use `MakeDataClient()`
We will need to wait until the changes from #7226 are in a release to update the quickstart.

The change is just that this:

https://github.com/googleapis/google-cloud-cpp/blob/659c66416eb9b5a84ec972a6ad58ac0627af4b6b/google/cloud/bigtable/quickstart/quickstart.cc#L34-L36

becomes:
```c++
cbt::Table table(cbt::MakeDataClient(project_id, instance_id), table_id);
```

oh and regenerate the README.md

### PR DESCRIPTION
**Pull request by dbolduc**
Fixes #7231

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7275)

### CODE REVIEW

File: ci/etc/vcpkg-config.ps1 [hunk 1]
```diff
@@ -15,4 +15,4 @@
 #
 # Common configuration parameters.
 #
-$env:VCPKG_RELEASE_VERSION="7e396645d919f463ef6f0f2ad6a8c5272c1f9b27"
+$env:VCPKG_RELEASE_VERSION="7e7bcf89171b7ed84ece845b1fa596a018e462f2"
```
Comment (coryan) [inline ci/etc/vcpkg-config.ps1:18]: nit: normally we split the upgrades to a separate PR, but this one is very small otherwise.

**Review by coryan — APPROVED** [review]

File: ci/etc/vcpkg-config.ps1 (hunk 1 above)
Comment (dbolduc) [reply ci/etc/vcpkg-config.ps1:18]: Ack
